scripts/rfdk_build_environment.sh

```diff
@@ -8,7 +8,7 @@
 # repository.
 
 # Make sure we're running from the root of the CDK repo
-if ! test -f package.json || ! grep '"name": "aws-rfdk-project",' package.json > /dev/null || ! grep '"private": true,' package.json > /dev/null
+if ! test -f package.json || ! grep '"name": "aws-rfdk",' package.json > /dev/null || ! grep '"private": true,' package.json > /dev/null
```

Comment:
This check is not conclusive anymore (the comment says it checks that we are at the repo root). Do we care? What if we instead look for the `.git` directory as well?

Reply:
It's still conclusive due to the next check in the chain -- looking for `private: true`. That line does not exist in the `packages/aws-rfdk/package.json`

Reply:
Aha, you're correct. All good then!